### DISCUSSION TITLE
Pull shark badge dissappeared

### DISCUSSION THREAD

**Discussion by edem-cyber** [community/community#110838] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Pull-shark badge disappeared off my profile. Why

**Comment by SebastienXia** [comment]
### 1) Look on ur profile in the achivements:
![Capture](https://github.com/community/community/assets/105004912/e4b0c80c-67db-426b-99cf-d67f4b8e8d45)

### 2) click on a badge:
![s](https://github.com/community/community/assets/105004912/6d197571-58f3-400a-9fe6-b81413725515)

### 3) Then click on the eye ta show the badge on your profile: 
![dd](https://github.com/community/community/assets/105004912/de27e577-8ab4-440f-98ba-b469f421bfbb)

**Reply by edem-cyber** [reply]
I've opened my profile achievements, and unfortunately, the Pull Shark badge is not listed there. Seems like some sort of GitHub bug.

**Comment by WizzzStark** [comment]
I encountered a similar problem, and after all other solutions, I found one method that resolved the issue:

1. Deselect the option for private contribution visibility and save your preferences.
2. Re-select the private contribution visibility option and update your preferences once more.
![image](https://github.com/community/community/assets/85120579/709c031b-0d87-491b-86c7-a8b557cd3549)


If the issue persists, visit the specified link changing {username} with your own account. If the badge is visible there, hide it, and then click to display the badge again.

https://github.com/{username}?tab=achievements

![image](https://github.com/community/community/assets/85120579/54b3d244-4b43-4ecc-9967-81500871ba2c)
